packages/ui/components/empty-screen/EmptyScreen.tsx

```diff
@@ -21,6 +21,7 @@ export function EmptyScreen({
   className,
   iconClassName,
   iconWrapperClassName,
+  limitWidth = true,
```

Comment:
This is to not limit the width of the empty screen which was looking better in the platform dashboard.